run_test.py

```diff
@@ -57,6 +57,7 @@
             'cudnn': 'cudnn51-cuda8',
             'nccl': 'nccl1.3.4',
             'protobuf-cpp': 'protobuf-cpp-3',
+            'ideep': 'ideep4py==1.0.1',
```

Comment:
`ubuntu14_py3` uses system Python (3.4), which is not supported by iDeep.

Reply:
Fixed.